Makefile

```diff
@@ -449,6 +449,7 @@ help:
 	@echo "  LOG_ALL=1                 Enables dump of the make process output, errors, and binary execution outputs into logs.txt"
 	@echo "  SYSTEM=sys_name           Includes the definitions for the requires modules and batch schedulers in the different systems"
 	@echo "                            This definitions must be in sys/SYSTEM.def. (Do not include the .def extension)"
+	@echo "  DEVICE_TYPE=dev_name      Specifies device type being used, either nvidia or amd, to change compiler flags as needed."
```

Comment:
I think we shouldn't add a new flag for this. AOMP already shows a way to discover AMD and nvidia-smi may be another option. However, this could be added in a different PR